ci/build_container/build_recipes/gperftools.sh

```diff
@@ -6,13 +6,16 @@ if [[ "${OS}" == "Windows_NT" ]]; then
   exit 0
 fi
 
-VERSION=2.7
-SHA256=1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9
+VERSION=fc00474ddc21fff618fc3f009b46590e241e425e
```

Comment:
Can you add a comment why this version is selected and add a TODO for going back to released version once new version is released?

Reply:
ack